src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx

```diff
@@ -1,43 +1,116 @@
-import React from "react"
-import { connect } from "react-redux"
+/**
+ * Load and display forecasts
+ */
+import { Point } from "@turf/helpers"
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)